client/src/Components/MyApplications/index.jsx

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import { Container } from 'react-bootstrap';
+import ApplicationCard from '../CommonComponents/OfferCard';
+import applications from '../utils/myApplications';
```

Comment:
you didn't push this file? so how can we test your PR?

Reply:
it is pushed in offer card component PR #61.
I'll push it again in this PR for review simplicity
thanks